aws/data_source_aws_region_test.go

```diff
@@ -89,197 +83,110 @@ func TestAccDataSourceAwsRegion_basic(t *testing.T) {
 			{
 				Config: testAccDataSourceAwsRegionConfig_empty,
 				Check: resource.ComposeTestCheckFunc(
-					testAccDataSourceAwsRegionCheck(resourceName),
-					resource.TestCheckResourceAttr(resourceName, "endpoint", "ec2.us-east-1.amazonaws.com"),
-					resource.TestCheckResourceAttr(resourceName, "name", "us-east-1"),
-					resource.TestCheckResourceAttr(resourceName, "description", "US East (N. Virginia)"),
+					resource.TestMatchResourceAttr(dataSourceName, "description", regexp.MustCompile(`^.+$`)),
```

Comment:
Why not?
```suggestion
					resource.TestMatchResourceAttrSet(dataSourceName, "description"),
```

Reply:
```
resource.TestMatchResourceAttr(dataSourceName, "description", regexp.MustCompile(`^.+$`)),
```

Ensures that the value has at least one character, while `TestCheckResourceAttrSet()` would accept and empty string.